src/Runtime/PyExecutionSession.cpp

```diff
@@ -162,6 +162,24 @@ std::vector<py::array> PyExecutionSession::pyRun(
   return outputPyArrays;
 }
 
+void PyExecutionSession::pySetEntryPoint(std::string entryPointName) {
+  setEntryPoint(entryPointName);
+  return;
+}
+
+std::vector<std::string> PyExecutionSession::pyQueryEntryPoint() {
```

Comment:
`ExecutionSession.{c|h}pp` use `queryEntryPoints()` so this should be `pyQueryEntryPoints()` to be consistent.

Reply:
Yes. Thanks!